plugins/woocommerce-admin/client/core-profiler/index.tsx

```diff
@@ -1144,15 +1146,37 @@ export const coreProfilerStateMachineDefinition = createMachine( {
 								completed: true,
 							} );
 						},
-						onDone: {
-							actions: 'redirectToWooHome',
-						},
+						onDone: [
+							{
+								target: 'sendToJetpackAuthPage',
+								cond: ( _context ) =>
+									_context.pluginsSelected.find(
+										( plugin ) => plugin === 'jetpack'
```

Comment:
I wonder if this is sufficient, I remember being redirected to Jetpack auth when Jetpack is installed but not connected and I selected WooCommerce Tax extension in OBW.

Reply:
However I'm not sure if that logic came from WC Core or Tax plugin

Reply:
Good suggestion. I'll check WooCommerce Tax and Shipping.

Reply:
The current OBW only considers [Jetpack](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/profile-wizard/index.js#L223). I still think it's a good idea to redirect to Jetpack auth when WooCommerce Tax or Shipping is selected. Let's implement that in a separate PR, though.

Reply:
Sounds good!

Reply:
Actually, I just tested in a JN with the current WooCommerce. It seems like if Jetpack is activated but not connected, it will still go to auth screen even though I didn't select any extension to install.

Do we need to adopt the same logic?

Reply:
As far as I understand, we should only redirect to Jetpack auth when Jetpack is selected. Let's check with @verofasulo ✋

Reply:
We've clarified this in the design call, the current logic with only going with the auth flow when Jetpack is selected 👍